app/assets/javascripts/repositories/renderers/view_renderers.js

```diff
@@ -208,18 +208,18 @@ $.fn.dataTable.render.RepositoryStockValue = function(data) {
 
 $.fn.dataTable.render.RepositoryConsumedStockValue = function(data) {
   let canManage = $('.repository-table').data('stock-consumption-editable');
-  if (data && data.value.consumed_stock_formatted) {
+  if (data && parseFloat(data.value.consumed_stock_formatted) > 0) {
```

Comment:
Why are you trying to compare to zero?

Reply:
In epic, if the user set consumption to 0, we need to render the default state like nothing set
`In case when user sets consumed amount to zero, confirming the modal returnes the stock display back “no consumption recorded yet” state.`

Reply:
So why than consumption is not set to nil on update?

Reply:
updated